doc/manuals/orion-api.md

```diff
@@ -3267,7 +3267,7 @@ A `http` object contains the following subfields:
 
 | Parameter | Optional | Type   | Description                                                                                   |
 |-----------|----------|--------|-----------------------------------------------------------------------------------------------|
-| `url`     |          | string | URL referencing the service to be invoked when a notification is generated. An NGSIv2 compliant server must support the `http` URL schema. Other schemas could also be supported. |
+| `url`     |          | string | URL referencing the service to be invoked when a notification is generated. It mus follow the `http` URL schema. |
```

Comment:
```suggestion
| `url`     |          | string | URL referencing the service to be invoked when a notification is generated. It must follow the `http` URL schema format. |
```

Reply:
Algo de `https` schema format would be valid?

Reply:
I think It is not needed to mention https